protocol/src/main/java/io/camunda/zeebe/protocol/record/value/EventType.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.protocol.record.value;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum EventType {
+
+  // Default
+  UNSPECIFIED(null),
```

Comment:
❌ I would propose to change this to `NULL`. Unspecified to me seems like it is an event but we haven't specified it's type, whereas null would mean that we it's not an event at all.